CHANGELOG.md

```diff
@@ -56,6 +56,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     custom security group mappings
 - **CUMULUS-2402**
   - Officially expose `sftp()` for use in `@cumulus/sftp-client`
+- **CUMULUS-2355**
+  - Add logic to disable `s3CredentialsEndpoint` based upon value for environment variable `DISABLE_S3_CREDENTIALS`. If set to "true", it will
+    disable the lambda.
```

Comment:
This isn't quite true - if the environment variable is set then it will disable the dispensing of S3 credentials from the endpoint, but the Lambda will still run. Also we should give the name of the Lambda for context

```suggestion
  - Added logic to disable `/s3Credentials` endpoint based upon value for environment variable `DISABLE_S3_CREDENTIALS`. If set to "true",  the endpoint will not dispense S3 credentials and instead return a message indicating that the endpoint has been disabled.
```

Reply:
Oh, that's true. I will update for clarity

Reply:
Also, it seems like documentation on this is called for somewhere, but we don't seem to have any public docs about managing this endpoint for integrators. Failing that, maybe adding a README to `packages/s3-credentials-endpoint` that calls it out would be good

Reply:
Actually, since this is an NGAP-only feature, we have a wiki page that would be worth updating instead:

https://wiki.earthdata.nasa.gov/display/CUMULUS/Cumulus+Direct+S3+Access

Reply:
[`fc51a9a` (#2099)](https://github.com/nasa/cumulus/pull/2099/commits/fc51a9a280c17bb4b8c023ed01d7e027f62336dc)

Reply:
Should I update the [wiki page](https://wiki.earthdata.nasa.gov/display/CUMULUS/Cumulus+Direct+S3+Access) after the feature goes out? I don't want to cause confusion by adding it prematurely. @markdboyd

Reply:
Ah good call. Yeah I guess we'll have to just update it after this change has been released